### PR TITLE
Adds group checkouts, requests, and fines

### PR DESCRIPTION
**Pull request by shelleydoljack**
Part of #159 
Screen Shot 2019-07-18 at 3 37 53 PM
Screen Shot 2019-07-18 at 3 37 34 PM

### Need to:
- [x] write tests
- [~] ~deal with Patron model class length (too long!)~
- [x] make the patron user's tab display `(P=<lastname>)`
- [x] remove the sponsor's data from member_list (to remove it from the group tab)
- [x] remove the current patron's data from the member_list (to remove it from the group tab)

### Split out to another ticket
- [~] add borrower name to group tab (put at top of expanded list item) for sponsor and proxies
- [~] styling around tab controls
- [~] display counts in the tabs (split this out to new ticket probably)

### CODE REVIEW

**Comment by shelleydoljack** [comment]
The currently logged in user's checkouts and the sponsor's checkouts are not included in the group list:
Screen Shot 2019-07-19 at 5 14 15 PM
Screen Shot 2019-07-19 at 5 14 05 PM

**Comment by shelleydoljack** [comment]
Changes display of proxy borrower name on tab per conversation with Jennifer:
Screen Shot 2019-07-22 at 11 05 11 AM

File: app/services/symphony_client.rb [hunk 1]
```diff
@@ -41,24 +41,28 @@ def session_token
     end
   end
 
-  # Symphony API requires some loooong strings, disabling LineLength for this method
-  # rubocop:disable Metrics/LineLength
+  ITEM_RESOURCES = 'item{*,bib{title,author},call{sortCallNumber,dispCallNumber}}'
+
+  PATRON_LINKED_RESOURCES = [
+    "holdRecordList{*,#{ITEM_RESOURCES}}",
+    "circRecordList{*,circulationRule{loanPeriod{periodType{key}}},#{ITEM_RESOURCES}}",
```
Comment (mejackreed) [inline app/services/symphony_client.rb]: I think that `renewFromPeriod` might have gotten dropped from this. I think this is needed to determine if an item is renewable and is still needed.

File: app/views/shared/_navigation.html.erb [hunk 2]
```diff
@@ -26,3 +26,16 @@
     </li>
   </ul>
 </nav>
+
+<% if patron.group? %>
+<nav class="container">
```
Comment (mejackreed) [inline app/views/shared/_navigation.html.erb]: Can we indent the `nav` element underneath the conditional?

File: spec/controllers/fines_controller_spec.rb [hunk 3]
```diff
@@ -42,7 +42,7 @@
       expect(get(:index)).to render_template 'index'
     end
 
-    it 'assigns a list of requests' do
+    it 'assigns a list of fines' do
```
Comment (mejackreed) [inline spec/controllers/fines_controller_spec.rb]: Nice catch

File: app/services/symphony_client.rb (hunk 1 above)
Comment (cbeer) [reply app/services/symphony_client.rb]: Good catch; I'll add that back.

**Pull request merged**